lib/tasks/osc.rake

```diff
@@ -74,6 +74,21 @@ namespace :osc do
     end
   end
 
+  def check_IDs_in_changes_file!
```

Comment:
I think it is better to create own task for, so projects like add-ons or modules which have less strict rules can disable it. Now it is hardcoded and everyone have to follow it. So something like `task check:changelog` and add it to dependecy chain at https://github.com/openSUSE/packaging_tasks/blob/master/lib/tasks/package.rake#L23 or later before doing `rake osc:commit`